test/starlark_tests/rules/linkmap_test.bzl

```diff
@@ -35,11 +35,7 @@ def _linkmap_test_impl(ctx):
     architectures = ctx.attr.architectures
 
     if not architectures:
-        platform_type = target_under_test[AppleBundleInfo].platform_type
-        if platform_type == "watchos":
-            architectures = ["i386"]
-        else:
-            architectures = ["x86_64"]
+        architecture = [ctx.fragments.apple.single_arch_cpu]
```

Comment:
@keith Looks like there's a typo in here (?) (should be plural) that makes it ignore all the assertions.

Reply:
oof thanks